apps/emqx_connector/src/emqx_connector.erl

```diff
@@ -134,7 +134,7 @@ post_config_update([?ROOT_KEY], _Req, NewConf, OldConf, _AppEnv) ->
             Result = perform_connector_changes([
                 #{action => fun emqx_connector_resource:remove/4, data => Removed},
                 #{
-                    action => fun emqx_connector_resource:create/3,
+                    action => fun emqx_connector_resource:create/4,
```

Comment:
Good catch!

Reply:
Since there does not seem to be any test case that catches this maybe we should add this but I think this can be done in another PR. Can you add a jira task for adding such a test case?

Reply:
Added a ticket: https://emqx.atlassian.net/browse/EMQX-11281